resolwe_bio/tests/workflows/test_rna_seq.py

```diff
@@ -132,15 +156,49 @@ def test_bbduk_star_featurecounts_workflow(self):
         self.run_process('workflow-bbduk-star-featurecounts-single', inputs)
         for data in Data.objects.all():
             self.assertStatus(data, Data.STATUS_DONE)
-        workflow = Data.objects.last()
-        self.assertFile(workflow, 'rc', 'feature_counts_rc_single.tab.gz', compression='gzip')
+        feature_counts = Data.objects.get(process__slug='feature_counts')
+        self.assertFile(feature_counts, 'rc', 'feature_counts_rc_single.tab.gz', compression='gzip')
+        multiqc = Data.objects.get(process__slug='multiqc')
+        self.assertFileExists(multiqc, 'report')
 
         inputs['preprocessing']['reads'] = paired_reads.id
         self.run_process('workflow-bbduk-star-featurecounts-paired', inputs)
         for data in Data.objects.all():
             self.assertStatus(data, Data.STATUS_DONE)
-        workflow = Data.objects.last()
-        self.assertFile(workflow, 'rc', 'feature_counts_rc_paired.tab.gz', compression='gzip')
+        feature_counts = Data.objects.filter(process__slug='feature_counts').order_by('-created')[0]
+        self.assertFile(feature_counts, 'rc', 'feature_counts_rc_paired.tab.gz', compression='gzip')
+        multiqc = Data.objects.filter(process__slug='multiqc').order_by('created')[1]
```

Comment:
You can use `order_by('-created')[0]`, to be consistent with other cases.

Reply:
Even better, rather use `latest('created')` or `last()`. Know that `last()` will order by primary key (`id`).